src/api/app/controllers/webui/patchinfo_controller.rb

```diff
@@ -158,6 +159,7 @@ def save
             Package.verify_file!(@package, '_patchinfo', xml)
           rescue APIException => e
             flash[:error] = "patchinfo is invalid: #{e.message}"
+            render action: 'edit_patchinfo', project: @project, package: @package
```

Comment:
Was this crashing the webui? I wasn't aware of any issues there. So I wonder if this is really necessary